app/Http/Controllers/Authentication/ForgetpasswordController.php

```diff
@@ -18,9 +20,11 @@ public function index ()
    }
 
    public function check(ForgetPasswordRequest $request)
```

Comment:
Class names should match class being imported and they should be in Pascal/title case, not camel case.